content/language/php/develop.md

```diff
@@ -0,0 +1,354 @@
+---
```

Comment:
Just one thought (so feel free to throw out/ignore)... a _lot_ of PHP developers will probably be more familiar with using MySQL/MariaDB, as it's the only supported database for Wordpress and the recommended db for Drupal. If you used MySQL, you can easily add another service to the Compose file for phpMyAdmin, which is yet another service PHP devs use quite often, many hosting services provide as part of hosting, etc. It could help show the ability to bolt on additional containers to the stack to make it easier to do development/debug data. Just a thought...

Reply:
ah, that is a great point! we currently use the postgres image as a commented out example in the generated compose file for init, but it wouldn't be hard to swap it out for a more relevant image. (i also don't see any issues with using a different image in the docs than whatever's currently used by init and will leave that up to craig.)